lesson03/index.js

```diff
@@ -0,0 +1,29 @@
+const jsTarget = document.getElementById("js-target");
```

Comment:
Q
Is it common to add js to variable names? I would like to know if you have a reference to this!

Reply:
一般的ではないので直したいですね

Reply:
@nakagawamai @kenmori 
All right.
jsTarget was not something that everyone could understand...

```suggestion
const ul = document.getElementById("js-target");
```
This it change to ul.